actioncable/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "actioncable",
+  "version": "5.0.0-rc1",
+  "description": "WebSocket framework for Ruby on Rails.",
+  "main": "dist/actioncable.js",
+  "files": [
+    "dist/*.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rails/rails.git"
+  },
```

Comment:
Believe this can be shortened to `"repository": "rails/rails"` (same shortcut syntax as `npm install`)